src/models/workspaces.ts

```diff
@@ -68,6 +68,7 @@ export const LearningLogWorkspaceModel = types
     document: DocumentModel,
     title: types.string,
     createdAt: types.number,
+    mode: types.maybe(types.undefined)
```

Comment:
I assume this is meant to be `types.maybe(types.string)`. I'm surprised MST let you do this.